bin/ci-stack.ts

```diff
@@ -7,12 +7,28 @@
  */
 
 import 'source-map-support/register';
-import { App } from '@aws-cdk/core';
+import { App, RemovalPolicy } from '@aws-cdk/core';
 import { CIStack } from '../lib/ci-stack';
 import { CIConfigStack } from '../lib/ci-config-stack';
+import { DeployAWSAssets } from '../lib/DeployAWSAssets';
 
 const app = new App();
 
-new CIConfigStack(app, 'CI-Config-Dev', {});
+const env: string = 'Dev';
 
-new CIStack(app, 'CI-Dev', { });
+const ciConfigStack = new CIConfigStack(app, `OpenSearch-CI-Config-${env}`, {});
+
+const ciStack = new CIStack(app, `OpenSearch-CI-${env}`, {
+  ecrAccountId: ciConfigStack.account,
+});
+
+new DeployAWSAssets(app, `OpenSearch-CI-Deploy-Assets-${env}`, {
+  removalPolicy: RemovalPolicy.DESTROY,
+  mainNodeAccountNumber: ciStack.account,
+  createEcrRepositories: true,
+  envName: env,
+  env: {
+    // public ECR repositories can only be created in us-east-1
```

Comment:
Can you also comment the issue that you found stating this fact so that we can track in future?